### ISSUE TITLE
(CTRL | CMD) + S still works with with autosave set

### ISSUE THREAD

**Issue by PP-Tom** [payloadcms/payload#9770]
### Describe the Bug

When you have autosave set, it's still possible to create a draft with CTRL + S or CMD + S, this creates unexpected behaviour such as deleting search indexes. 

### Link to the code that reproduces this issue

pnpx create-payload-app@latest -t blank

### Reproduction Steps

1. Enable autosave on a collection.
2. Press CTRL + S or CMS + S

### Which area(s) are affected? (Select all that apply)

plugin: search, Not sure

### Environment Info

```text
Payload: 3.4.0
NodeJS: 22
NextJS: 15.0.3
```

**Comment by paulpopus** [comment]
I don't think this issue is in particular with the hotkey to save while autosave is enabled. The search plugin issue can happen without autosave as well so I'll be fixing that by adding an additional check for a published version and only then removing the search doc if no published versions exist.

If you see any other issues regarding these flows let us know and we can handle that directly.
